diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -79,8 +83,10 @@ DiffDriveController::init(const std::string & controller_name)
 
     node->declare_parameter<std::string>("odom_frame_id", odom_params_.odom_frame_id);
     node->declare_parameter<std::string>("base_frame_id", odom_params_.base_frame_id);
-    node->declare_parameter<std::vector<double>>("pose_covariance_diagonal", {});
-    node->declare_parameter<std::vector<double>>("twist_covariance_diagonal", {});
+    node->declare_parameter<std::vector<double>>("pose_covariance_diagonal", std::vector<double>());
+    node->declare_parameter<std::vector<double>>(
+      "twist_covariance_diagonal",
+      std::vector<double>());
```

Comment:
Propose to reduce the number of lines...

```suggestion
    node->declare_parameter<std::vector<double>>(
      "twist_covariance_diagonal", std::vector<double>());
```